lib/angular-evaporate.js

```diff
@@ -147,6 +157,11 @@
                       (100 - that.$percent) / that.$percent *
                       (now - that.$started) / 1000
                     );
+                    var progress = (arg * 100).toFixed(2);
+                    var delta = (new Date() - that.$started) / 1000;
+                    var loaded = (progress * that.sizeBytes) / 100;
+                    var totalSentBySecond = loaded / delta;
+                    that.$speed = readableFileSize(totalSentBySecond) + '/s';
```

Comment:
I can provide a more detailed snippet of my proposition, or if you don't have time to dig more into the code, we can just merge this into the `dev` branch (not `master`) and I'll do it myself.